spec/stat_tracker_spec.rb

```diff
@@ -117,46 +117,59 @@
   end
 
   describe '#lowest_scoring_visitor' do
-    it "name of team that scored lowest average goals while away" do
+    xit "name of team that scored lowest average goals while away" do
       expect(@stat_tracker.lowest_scoring_visitor).to eq "San Jose Earthquakes"
     end
   end
     
   describe 'offense' do
-    it 'has #best_offense' do
+    xit 'has #best_offense' do
       expect(@stat_tracker.best_offense).to eq "Reign FC"
     end
     
-    it "#worst_offense" do
+    xit "#worst_offense" do
       expect(@stat_tracker.worst_offense).to eq "Utah Royals FC"
     end
   end
     
   describe 'lowest_scoring_home_team' do
-    it "name of team that scored lowest average goals while home" do
+    xit "name of team that scored lowest average goals while home" do
       expect(@stat_tracker.lowest_scoring_home_team).to eq "Utah Royals FC"
     end
   
-    it "#highest_scoring_visitor" do
+    xit "#highest_scoring_visitor" do
     expect(@stat_tracker.highest_scoring_visitor).to eq "FC Dallas"
     end
 
-    it "#highest_scoring_home_team" do
+    xit "#highest_scoring_home_team" do
     expect(@stat_tracker.highest_scoring_home_team).to eq "Reign FC"
     end
   end
 
   describe "Tackles" do
-    it "#most_tackles" do
+    xit "#most_tackles" do
       expect(@stat_tracker.most_tackles("20132014")).to eq "FC Cincinnati"
       expect(@stat_tracker.most_tackles("20142015")).to eq "Seattle Sounders FC"
     end
 
-    it "#fewest_tackles" do
+    xit "#fewest_tackles" do
```

Comment:
are these all meant to be skipped?

Reply:
My bad! I'll unskip them on the next push

Reply:
no worries at all! 
Great work on starting iteration 4!